recipes-app/iot2050-eio-manager/iot2050-eio-manager_0.1.bb

```diff
@@ -59,6 +59,8 @@ DEBIAN_DEPENDS = "python3, python3-grpcio, python3-dotenv, python3-jsonschema, \
 python3-yaml, python3-bitstruct, python3-libgpiod, libflashrom1, libflashrom-dev, \
 python3-progress, python3-psutil, libfuse2, "
 
+DEBIAN_BUILD_DEPENDS = "libfuse2, libgpiod-dev"
+
```

Comment:
You are missing `DEBIAN_DEPENDS`.

Reply:
And, strictly spoken, you only need libgpiod2 (for both).

Reply:
libfuse2 is in line 60

Reply:
Ah, sorry. And `python3-libgpiod` likely pulls `libgpiod2` into the runtime deps. OK, then just make it `libgpiod2` for the build-time deps.

Reply:
And add some explanation: dh_shlibdeps checks for the library dependencies of the pre-built binary during installation, and those aren't fulfilled unless we add those libs as build-time dependencies.

Reply:
OK, refined it